.github/ISSUE_TEMPLATE/enhancement-template.yml

```diff
@@ -0,0 +1,55 @@
+name: Enhancement Report
```

Comment:
why would this table be on top of the form?

Reply:
The table is only visible in the preview, not the actual form.